build-info-client/src/main/java/org/jfrog/build/client/PreemptiveHttpClientBuilder.java

```diff
@@ -181,6 +181,7 @@ protected HttpClientBuilder createHttpClientBuilder() {
                 .custom()
                 .setSocketTimeout(timeoutMilliSeconds)
                 .setConnectTimeout(timeoutMilliSeconds)
+                .setConnectionRequestTimeout(timeoutMilliSeconds)
```

Comment:
It looks like the value we set for `timeoutMilliSeconds` is one second? Is it not?
This seems like a very short timeframe. I suggest changing it to 30 seconds, and apply it to 3 timeout values.

Reply:
It is not 1 second, the default is 5 minutes